user_service/infra/db.go

```diff
@@ -15,20 +14,20 @@ func DbConnection() *sql.DB {
 			id SERIAL PRIMARY KEY,
 			username VARCHAR(50) NOT NULL,
 			email VARCHAR(100) NOT NULL,
-			password VARCHAR(100)
+			password VARCHAR(100),
```

Comment:
Email and username should be unique.
Also I would recommend you move this migrations.